Makefile

```diff
@@ -0,0 +1,20 @@
+# perf-analyzer Makefile
+
+PACKAGE=github.com/openshift-scale/perf-analyzer
+PACKAGE_SCRAPER=$(PACKAGE)/cmd/scraper
+PACKAGE_SCRAPER_BIN=$(lastword $(subst /, ,$(PACKAGE_SCRAPER)))
+PACKAGE_COMPARE=$(PACKAGE)/cmd/compare
+PACKAGE_COMPARE_BIN=$(lastword $(subst /, ,$(PACKAGE_SCRAPER)))
+ENVVAR=GOOS=linux CGO_ENABLED=0 GOARCH=amd64 
+OUT_DIR=_output
+
+all: scraper compare
+
+scraper:
```

Comment:
Would be nice to have a list of source dependencies so that if any of the sources change, the target only rebuilds on that.

Reply:
Great thanks!